tests/test_create_consensus_network.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/env/bin python3
+
+import unittest
+import filecmp
+import sys
+import os
+import shutil
+from SJARACNe.bin.create_consensus_network import create_consensus_network as cn
+from SJARACNe.bin.create_consensus_network import create_enhanced_consensus_network as ecn
+from SJARACNe.bin.create_consensus_network import uprob
+
+class TestConsensusNetwork(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        self.path = './test_answers'
+        if self.path is None:
+            os.mkdir('./test_answers')
+        cn('./tests/inputs/adjmat_dir', 0.05, './test_answers')
+
+    
+    def test_consensus_network_3col(self):
+        self.assertTrue(filecmp.cmp('./tests/answerkey/consensus_network_3col_.txt', './test_answers/consensus_network_3col_.txt'))
+        
+    def test_bootstrap_info(self):
+        self.assertTrue(filecmp.cmp('./tests/answerkey/bootstrap_info_.txt', './test_answers/bootstrap_info_.txt'))
+
+    def test_parameter_info(self):  
+        #delete last line of parameter info, as it will not match with answer key
```

Comment:
Is it possible to add an extra line to answer key to match with test answer?

Reply:
The last line of the parameter file is location of the 3col file (Output Network). The only problem with this would be that using a temporary directory and I don't get to pick it's name. We could add the line "Output Network: tempdir.name/3colFile.txt" every time, but then we'd have to remove the last line anyway since the tempdir will have different name attributes every time test is run.